src/removeStopwords.js

```diff
@@ -0,0 +1,38 @@
+// SUGGESTIONS: refacture ino shouldDisplay
+// and this function take a word and the language, then it returns true or false, and this function should be used wherever you are trying to display the words
+
+import {  STOPWORDS } from './constants';
+
+const filterWords = (wordCountObject, wordsToFilterOut) => {
+  for (var key in wordCountObject) {
+    var containsWord = (wordsToFilterOut.indexOf(key) > -1);
+    if (containsWord) {
+      delete wordCountObject[key];
+    }
+  }
+  return wordCountObject;
+}
+
+export const removeStopwords = (wordCountObject, language) => {
+  var stopwords = STOPWORDS[language];
+  var filteredObject = Object.assign(wordCountObject);
```

Comment:
What does this line do?

Reply:
This creates a new object. So that we don not mutate the original object, but instead have two separate ones.

Reply:
but this is not what this line does. check `Object.assign` method documentation.

Reply:
> The Object.assign() method is used to copy the values of all enumerable own properties from one or more source objects to a target object. It will return the target object.

Check out this codepen. Here I create a new object and then manipulate it without the first object being mutated.
https://codepen.io/mowolf/pen/pKjGyb?editors=0012